ipynb/tests/Generic_EAS_Tests.ipynb

```diff
@@ -0,0 +1,533 @@
8078459999996994, 6.8078539999987697], [6.8086209999983112, 6.8086289999992005], [6.8510799999985466, 6.8511059999982535], [6.8526979999987816, 6.8527389999999286], [6.8527579999990849, 6.8527849999991304], [6.8551359999983106, 6.855160999999498], [6.8591469999992114, 6.8591979999982868], [6.8607439999996132, 6.8607609999999113], [6.8607899999988149, 6.8608009999989008], [6.863058999999339, 6.8630699999994249], [6.8633029999982682, 6.8633139999983541], [6.8916709999994055, 6.891684999998688], [6.8993519999985438, 6.8993679999985034], [6.9029079999982059, 6.9029149999987567], [6.911099999999351, 6.9111629999988509], [6.9128249999994296, 6.9128549999986717], [6.9128799999998591, 6.9129059999995661], [6.9149209999995946, 6.914956999999049], [6.9166409999997995, 6.9166499999992084], [6.9166839999998047, 6.916693999999552], [6.9236839999994118, 6.9236949999994977], [6.9255159999993339, 6.9255229999998846], [6.966919999998936, 6.9669359999988956], [6.9673059999986435, 6.9673279999988154], [6.969649999999092, 6.9696749999984604], [6.9697459999988496, 6.9697799999994459], [6.9699469999995927, 6.9699789999995119], [6.9712859999999637, 6.9713279999996303], [6.9730279999985214, 6.9730559999989055], [6.9731469999987894, 6.9731609999998909], [7.054573999999775, 7.0546039999990171], [7.054932999999437, 7.0549499999997352], [7.0563729999994393, 7.05640399999902], [7.0565759999990405, 7.0566099999996368], [7.0579209999996237, 7.0579429999997956], [7.0590329999995447, 7.0590569999985746], [7.061491999998907, 7.061506999998528], [7.0616989999998623, 7.0617149999998219], [7.089070999998512, 7.0890829999989364], [7.0909130000000005, 7.0909209999990708], [7.1118179999994027, 7.1118249999999534], [7.1271179999985179, 7.127145999998902], [7.1289159999996627, 7.1289569999989908], [7.1290449999996781, 7.1290729999982432], [7.1311389999991661, 7.1311669999995502], [7.1334329999990587, 7.1334889999998268], [7.1335029999991093, 7.1335199999994074], [7.1351799999993091, 7.1351979999999458], [7.1353299999991577, 7.1353419999995822], [7.1392589999995835, 7.1392769999984012], [7.1751179999992019, 7.1751319999984844], [7.1753509999998641, 7.1753609999996115], [7.1855259999992995, 7.185550999998668], [7.1870129999988421, 7.1870339999986754], [7.1873479999994743, 7.1873819999982516], [7.1874099999986356, 7.187432999999146], [7.188732999999047, 7.1887699999988399], [7.1901979999984178, 7.1902069999996456], [7.1902429999991, 7.1902529999988474], [7.2023279999993974, 7.2023379999991448], [7.2029539999984991, 7.2029599999987113], [7.205298999999286, 7.2053079999986949], [7.2077349999999569, 7.2077469999985624], [7.2151149999990594, 7.2151239999984682], [7.2404409999999189, 7.2404649999989488], [7.242086999998719, 7.242127999999866], [7.2422169999990729, 7.2422439999991184], [7.3629449999989447, 7.3629849999997532], [7.3646639999988111, 7.3646809999991092], [7.3649589999986347, 7.3649699999987206], [7.4134959999992134, 7.4135049999986222], [7.4150779999999941, 7.4150989999998274], [7.4
```

Comment:
It would be more clear if these would print a success message when the test is passed as there is no output from it otherwise. The same for test_task_placement.